src/borkdude/deps.clj

```diff
@@ -430,21 +430,23 @@ For more info, see:
 
 (def java-exe (if windows? "java.exe" "java"))
 
+(def ^:private java-cmd
```

Comment:
Making this a top-level constant isn't working correctly with GraalVM native-image as it will capture the Java command of the compilation environment. Please make it a function or revert this change.

Reply:
Converted it into a fn since it is used in one of the tests, thanks.